releases/data-center/syskit-point-2024-2/point-2024-2-45-release-note.md

```diff
@@ -0,0 +1,109 @@
+---
+description: >-
+  This article lists improvements and bug fixes in Syskit Point version 2024.2.45
+---
+
+# Syskit Point 2024.2.45
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2024.2.45
+
+**Build number:** 79
+
+**Release date:** April 10, 2024
+
+## Get the Latest Version
+
+* **New to Syskit Point?** Get the latest Syskit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit\_point).\
+  Find all steps described in detail in the [deployment article](../../../set-up-point-data-center/deployment/deploy-syskit-point.md).
+* **Already using Syskit Point?** Upgrade to the latest Syskit Point version.\
+  [![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FARMTemplates%2FPointUpdateDeploy%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade Syskit Point in the [upgrade article](../../../set-up-point-data-center/deployment/upgrade-syskit-point.md).
+
+## Improvements & Bug Fixes
+
+{% hint style="warning" %}
+**Please note!**\
+The upgrade to this Syskit Point version may take longer to complete, so we kindly ask for your patience while upgrading. 
+
+After upgrading to the latest version, sites need to be synced for data to show on the Storage Metrics report due to performance optimization changes introduced.
+The first site sync after the upgrade will be a full site sync instead of a partial one, which is required for the optimization changes to take effect.
+{% endhint %}
+
+* **Storage Management gets new improvements!**
```

Comment:
budući da se radi o major releaseu, uveo bi ##New Features sekciju i tu dodao storage viewove, clean up akciju (not sure kada je dodano ono za setup site quote, to mislim da fali, neznam zašto)